getting-started/templates/pilot-sizing.yaml

```diff
@@ -163,7 +163,21 @@ dataframeservice:
   #   zookeeper:
   #     count: 1
 
+dynamicformfields:
+  resources:
+    requests:
+      cpu: "125m"
 
+      memory: "160Mi"
+    limits:
+      memory: "160Mi"
+  autoscaling:
+    enabled: true
+    minReplicas: 1
+    maxReplicas: 3
```

Comment:
For the pilot sizing, we have constrained the maxReplicas to 1 or 2 in most situations.  Is there justification in a pilot size deployment why this would need to autoscale to 3 pods?

Reply:
No, there is no rationale for a specific value.
I will reduce to 1.